### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by Praveit** [cohenerickson/Velocity#44]
### What Happened

https://user-images.githubusercontent.com/119162325/233830078-29ed90c1-c94f-421c-9429-205f5ae07ad8.png 
Thats the error I'm getting and apparently it's because the Ultraviolet proxy version being used is outdated, could you update the files to the newer version and you also need the tomphttp bare server node to be updated and I tried to do it myself but I'm still new to all this so I can't really do it, if you could do it you would have support for website logins which would be great and otherwise Velocity is extremely fast you chose a great proxy and there are plenty of customizable options, just one more request: Extension support from chrome webstore?

### Expected Behavior

```text
Not this, I thought I would be able to log in to outlook
```

### Steps to Reproduce

```text
Just go to outlook.com and try logging in
```

### Other Information

```text
Not really everything is explained in the first one
```

### Checklist

- [X] I have read and followed the contributing guidelines

**Comment by ExtremeCoder991** [comment]
Having the same issue

**Comment by cohenerickson** [comment]
fixed [7ba21b7](https://github.com/cohenerickson/Velocity/commit/7ba21b7f80d5a822ff6eab139e66dabb26183d7c)
